src/System.Runtime.Extensions/src/System/IO/BufferedStream.cs

```diff
@@ -49,7 +49,7 @@ public sealed class BufferedStream : Stream
         private const int MaxShadowBufferSize = 81920;  // Make sure not to get to the Large Object Heap.
         private const int DefaultBufferSize = 4096;
 
-        private Stream? _stream;                            // Underlying stream.  Close sets _stream to null.
+        private Stream _stream;                            // Underlying stream.  Close sets _stream to null.
```

Comment:
If close sets _stream to null, why isn't it nullable? Or is the comment wrong?

Reply:
I see the comment is not null - can you explain their reasoning?

Reply:
Yeah I think I'll leave this as nullable and just bang away `return _stream!` on UnderlyingStream. We didn't want to make UnderlyingStream nullable just because we set `_stream` to null on Close/Dispose. 

We agreed that, even though the annotation for `UnderlyingStream` will be non-null, if you ever get a null out of it is because you're doing an invalid operation which is trying to use a closed/disposed stream.

Reply:
Fixed.